Tests/L0/Gradle/data/multimodule/module-one/module-one.iml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is test data - I've added a few files here and there to make the tests more realistic. Only the xml and html reports matter.